.github/workflows/pull-request.yml

```
@@ -272,35 +272,71 @@
-  #         node-version: '22.x'
-
-  #     - name: resolve dependency
-  #       run: npm install -g @graphql-inspector/cli
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '22.x'
+
+      - name: resolve dependency
+        run: npm install -g @graphql-inspector/cli@latest
       
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)